lib/ok.ex

```diff
@@ -161,22 +161,39 @@ defmodule OK do
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
       ...>   b <- safe_div(a, 2)
+      ...>   OK.success a + b
+      ...> end
+      {:ok, 6.0}
+
+      iex> OK.with do
+      ...>   a <- safe_div(8, 2)
+      ...>   b <- safe_div(a, 2)
       ...>   {:ok, a + b}
       ...> end
       {:ok, 6.0}
 
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
+      ...>   b = 2.0
+      ...>   OK.success a + b
```

Comment:
I just know about it because I had a ton of warnings to fix. I've still only fixed my ib_gib engine and not my web_gib phoenix app, which still has an absurd amount of warnings. :sweat_smile: